### PR TITLE
Created web bootcamp

### PR DESCRIPTION
**Pull request by thecodechaser**
## Steps taken

- Cloned the repository
- Created a new branch from the main branch
- Created a gitignore file and added node_modules
- Set up necessary linters and tools
- Created markup for index page
1. Header section
2. Hero section
3. Main section
4. Featured tutors section
5. partners, contact form sections
6. footer section
- Build the featured tutors section dynamically using javascript
- Styled the Index page for mobile
- Styled the index page for desktop using media queries
- Created navbar menu for mobile devices using javascript events
- Applied transitions and animations effects to index page's elements
- Created markup for about page
1. Header section
2. Hero section
3. Main section
4. partners, contact form sections
5. footer section
- Styled the about page for mobile
- Styled the about page for desktop using media queries
- Created navbar menu for mobile devices using javascript events
- Applied transitions and animations effects to about page's elements
- Fix linter issues

### Javascripts use
- Created featured tutors section dynamically using javascript
- Used arrays, objects to store and get data
- Created navbar menu for mobile using javascript events

### Animations and transitions effects
- Header logo
- Hero section content
- Hero section images
- Main section
- Featured tutors
- Footer sections
- Images

## Video Demo
[Loom Video]()

### Home Page
Screenshot (75)

Screenshot (76)

Screenshot (77)

Screenshot (78)

### About page
Screenshot (79)

Screenshot (80)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            </li>
+            <li>
+            <a class="mobile-menu-item" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+        </div>
+      </nav>
+      </div>
+    </header>
```
Comment (omar-labana) [inline about.html]: - [ ] The hamburger menu should be on the left side, please move it so it matches the design.
- [ ] The `header` is not identical in both pages, kindly update it.

Reference Design
image

Your Styling => `320px`
image
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        <div class="desktop-nav">
+          <ul class="desktop-menu-items">
+            <li>
+            <a class="desktop-menu-item" href="index.html">Home</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#">Program</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#">Tutors</a>
```
Comment (omar-labana) [inline about.html:42]: - [ ] The `header` is broken on the smallest desktop screen, please update it so it look perfect on all screen sizes.

Reference Design
image

Your Styling => `786px`
image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+        <button type="submit" class="button contact">Submit</button>
+        </form>
+      </section>
+
+      <!-- #### copyright-section #### -->
+      <section class="copyright-section">
+        <img alt="copyright logo" src="images/header-logo.png" class="copyright-logo">
+        <p class="copyright-details">2021 TheCodeChaser All Rights Reserved</p>
+      </section>
+    </footer>
```
Comment (omar-labana) [inline about.html]: - [ ] `partner` section should be horizontally centered. It also should be hidden on the `about` page.
- [ ] `footer` text has 2.28:1 contrast ratio, kindly update the font color so it becomes more readable at least `4.5:1`.
- [ ] Please add more height/padding to the `footer` in on desktop screen to match the design.
- [ ] Kindly make sure that the `footer` along with the `partners` section are centered horizontally.

Reference Design
image

image


Your Styling => '786px'
image

image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+        <div class="headline-img-container">
+          <div>
+        <h2 class="headline-heading3">NEXT COHORT STARTING IN 15 DAYS</h2>
+        <p class="headline-details2">Book your enrolment now</p>
+        </div>
+        <img class="hero-img" alt="hero image" src="images/hero-bg.png">
+        </div>
+        <p class="headline-details">Please contact us per email for any further quastions about TheCodeChaser bootcamp</p>
+        <p class="email">contact@thecodechaser.com<i class="fas fa-chevron-right"></i></p>
+      </section>
```
Comment (omar-labana) [inline about.html]: - [ ] The `hero` section (first section of the home page) should fill the whole screen view on `desktop` sized screens, the `font-size` is a bit smaller compared to the design. Kindly update the `hero` section to match the design.
- [ ] _[OPTIONAL]_ Pleas add a background image to the hero section that has a similar color scheme to the one found in the design.
- [ ] Content inside the `hero` section should be aligned a bit to the left on `desktop` screens.
Reference Design
image
Your Styling => '786px'
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+          </div>
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon5.png">
+            <h3 class="highlight-item-heading">Help</h3>
+            <p class="highlight-item-details">Our staff is available for you all the time to help you with quastions.</p>
+          </div>
+        </div>
+        <button type="button" class="button join-us">Enroll now and confirm your spot</button>
+        <a class="highight-end" href="#">SEE THE WHOLE PROGRAM</a>
+      </section>
```
Comment (omar-labana) [inline index.html]: - [ ] Kindly update the used tags within this section to use an `<ul;>` along with `<li>`s and `<article>` (for each card) to give a semantic meaning for the section. The usage of `div` tags is a bad practice.
- [ ] Please make sure to horizontally center this `section` to match the design.

Reference Design
image

Your Styling => `786`
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+      <!-- #### featured-courses-section #### -->
+      <section class="featured-tutors" id="tutors">
+      </section>
```
Comment (omar-labana) [inline index.html:143]: - [ ] Please make sure to horizontally center this `section` on `desktop` screens to match the design.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @thecodechaser, 

Excellent work so far! This is by far one of the best projects I have reviewed on this module 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️

- [ ] The video presentation is missing a feature code highlight; I would suggest recording a new one highlighting a piece of code that you find interesting or proud of making along with an overview of the project.

- [ ] Please make sure to follow the design guidelines for fonts used `[Cocogoose, Lato]` and the color scheme `[#272A31, #EC5242, #D3D3D3]`

- [ ] The `Live Demo` link in the README is broken, please make sure to deploy your project using GitHub Pages.

- [ ] The `Get Started` section in the README file is missing, I would suggest adding a numbered list with all the instruction necessary to run the project.

- [ ] Kindly make sure to test both of the pages in `320px` for mobile screens and `786px` for desktop screens. 

- [ ] Both pages should contain an `h1` header title about the page.

- [ ] Kindly make sure to apply changes required in both page styles, if you need any help kindly reach out on Slack, I would be more than happy to help.

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+    </header>
+
+    <!-- #### main-section #### -->
+    <main>
+
+      <!-- #### hero-section #### -->
+      <section class="hero-section">
+        <h2 class="headline-heading1">"Hello! technology geeks"</h2>
+        <h1 class="headline-heading2">
+          BECOME A PROFESSIONAL REMOTE WEB DEVELOPER
```
Comment (afanimartin) [inline about.html:89]: - This heading should be much bigger than what it is right now. This was already pointed out by the previous reviewer and please also align the contents of the hero section to the left. Right now it appears centered on desktop screens.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @thecodechaser, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**